src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+
+import logo from 'assets/logo.png';
+import PATHS from 'constants/paths';
+import { LOCAL_STORAGE_KEYS } from 'constants/general';
+import LocalStorageService from 'services/LocalStorageService';
+
+import styles from './styles.module.scss';
+
+function Navbar() {
+  const { t } = useTranslation();
+  const history = useHistory();
+
+  const handleLogout = () => {
+    LocalStorageService.removeValue(LOCAL_STORAGE_KEYS.client);
+    LocalStorageService.removeValue(LOCAL_STORAGE_KEYS.session);
+    LocalStorageService.removeValue(LOCAL_STORAGE_KEYS.uid);
+    history.push(PATHS.login);
```

Comment:
what about removing headers?
i would put the LS remove values as an global util exported from LocalStorageService to possible reuse. Also i would call `removeHeaders` or something like that, i think that could be a global utils too (exported from config/api/utils`.